test/parallel/test-tls-parse-cert-string.js

```diff
@@ -36,3 +38,17 @@ const tls = require('tls');
   const invalidOut = tls.parseCertString(invalid);
   assert.deepStrictEqual(invalidOut, {});
 }
+
+{
+  const regexp = new RegExp('^\\(node:\\d+\\) [DEP0073] DeprecationWarning: ' +
+                            'tls\\.parseCertString is deprecated$');
+
+  // test for deprecate message
```

Comment:
You can use `common.expectWarning()` to check for this without having to hijack stderr